pkg/backends/tcp.go

```diff
@@ -96,10 +96,6 @@ func (b *TCPListener) GetAddr() string {
 	return b.li.Addr().String()
 }
 
-func (b *TCPListener) GetCost() string {
-	return b.li.Addr().String()
-}
-
```

Comment:
I don't think we should remove GetCost but fix it to get the cost

something like:
```
func (b *TCPListener) GetCost() float64 {
	return cfg.Cost
}
```

Reply:
@PabloHiro You're correct, the GetCost that is actually used is TCPListenerCfg GetCost. We can delete this.